build/BuildDefaults.props

```diff
@@ -2,11 +2,12 @@
   <PropertyGroup>
     <CLITargets Condition=" '$(CLITargets)' == '' ">Prepare;Compile;Test;Package;Publish</CLITargets>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <HasAdditionalSharedFramework Condition="'$(Rid)' != 'linux-x64'">true</HasAdditionalSharedFramework>
+    <HasAdditionalSharedFramework Condition=" '$(HasAdditionalSharedFramework)' == '' ">false</HasAdditionalSharedFramework>
```

Comment:
We had two properties to begin with because these are 2 different things:

1. Does the current build even HAVE an additional shared framework?  Which for linux-x64, it doesn't.  And any new distro we support like alpine.
2. Even if we have the additional shared fx, should it get included?